mpas_analysis/ocean/sst_timeseries.py

```diff
@@ -88,17 +107,22 @@ def sst_timeseries(config, streamMap=None, variableMap=None):
 
     if ref_casename_v0 != 'None':
         print '  Load in SST for ACMEv0 case...'
+        inRefDate = '0001-01-01 00:00:00'
+
         infiles_v0data = '{}/SST.{}.year*.nc'.format(indir_v0data,
                                                      ref_casename_v0)
```

Comment:
No, I don't think so but you're right that there's definitely a problem here.  The observations and v0 results do have their own `inRefDate` and `outRefDate`, neither of which necessarily correspond to those of the simulation being analyzed.

As an example, let's take @akturner's run starting at 1959-01-01.  You'd want that to be both `inRefDate` and `outRefDate` for the MPAS data loaded in above, so you'd want to set `yr_offset = 0`.  But v0 assumes that `inRefDate = 0001-01-01` and `outRefDate = 1850-01-01`, so there's currently no way to know that from the config file.  My feeling is that these two ref dates should be config options, and that there should be a subsection in the config file for each set of observations that has its own ref dates.  Something like:
```
[oceanObservations]
# directory where observations are stored
basedir = /global/project/projectdirs/acme/observations/Ocean
# subdirectories for specific analysis
sst_subdir = SST
sss_subdir = SSS
mld_subdir = MLD
# reference date of the input data in format YYYY-MM-DD or YYYY-MM-DD hh:mm:ss
inRefDate = 0001-01-01
# corresponding reference date used in the analysis
outRefDate = 1850-01-01
```
I would think that approach would be a lot cleaner and clearer to someone reading the code.  I think it would be really non-intuitive if someone has to specify `yr_offset = 1849` because the observations and/or previous results expect that, but then it gets ignored for the current run.